.github/workflows/gotest.yml

```diff
@@ -30,8 +30,16 @@ jobs:
           make check-commits
         working-directory: code
 
+  precommit:
```

Comment:
this seems to do the "same" thing as commit message (checking validity of a commit) and it even had the same icon! :stuck_out_tongue:  would it make sense to combine it?

Reply:
I'm happy to propose it in a follow up :)

Reply:
I actually had it combined in my initial version but I could not figure out how to run action in different path (`./code`) instead of root, so I did split it. Feel free to refactor.

Reply:
I've proposed it in https://github.com/RHEnVision/provisioning-backend/pull/745 but I'm actually not sure if its much better, but have a look and we can decide there :)